starters/basic-starter/components/taxonomy/taxonomy-term.tsx

```diff
@@ -0,0 +1,43 @@
+import { Layout } from '../layout';
+import { TaxonomyArticle } from './taxonomy--article_type';
+import { TaxonomyPerson } from './taxonomy--person_type';
+import { TaxonomyEvent } from './taxonomy--event_type';
+import { TaxonomyPlace } from './taxonomy--place_type';
+import * as React from 'react';
+
+export function TaxonomyTerm({ node_list, menus, taxonomy_term, type }) {
+  return (
+    <Layout title={taxonomy_term} menus={menus}>
+      {type === 'taxonomy_term--article_type' &&
+        node_list.every(
```

Comment:
I think we could actually potentially remove this check and move this handling to the individual term components. This would probably simplify this enough so that we could remove this component completely and move it back to the slug.